agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -1518,25 +1518,25 @@ bool main_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
             LOG(DEBUG) << "slave is backhaul manager, updating";
             m_pBackhaulManagerSlave = sd;
 
-#if !defined(BEEROCKS_LINUX) && !defined(BEEROCKS_TURRIS_OMNIA) && !defined(BEEROCKS_NIGHTHAWK)
-            // Start ARP monitor
-            if (enable_arp_monitor) {
-                if (!init_arp_monitor()) {
-                    LOG(ERROR) << "can't start ARP monitor";
-                    return false;
+            if (bpl_cfg_monitoring_supported()) {
+                // Start ARP monitor
+                if (enable_arp_monitor) {
+                    if (!init_arp_monitor()) {
```

Comment:
It would make more sense to me if this were implemented by just having an implementation of `init_arp_monitor` that returns true without doing anything. However, since init_arp_monitor is implemented in platform_manager instead of in bpl, that's not possible, so OK (for now).

Reply:
It's also possible to extend the return value of the `bpl_arp_mon_start()` function (called internally from `init_arp_monitor()`)
For example, 0 on success, -1 on error and -2 if not supported/implemented.

Reply:
Yes, I like that!

Reply:
you 2 are making me work harder :)

OK - we'll go with this approach, but know that it will require some compromises - I will update bpl_err.h to include -1 and -2 for BPL_ERR_UNKNOWN (-1) and BPL_ERR_OPERATION_NOT_SUPPORTED (-2). I will not however replace all -1 to BPL_ERR_UNKNOWN in all of the BPL :)

Reply:
See #508